awsauth/roundtripper_test.go

```diff
@@ -0,0 +1,89 @@
+package awsauth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	dphttp "github.com/ONSdigital/dp-net/v2/http"
+	"github.com/ONSdigital/dp-net/v2/http/httptest"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
I guess we don't need to prefix this import with a dot ?

Reply:
We do